lib/routes/token.js

```diff
@@ -19,6 +20,7 @@ function generateToken(code) {
 }
 
 function toToken(authAt, _, token) {
+  activityEvent('token', hex(token.userId), hex(token.clientId), null /* server-to-server */);
```

Comment:
Likewise, at a high level this event is part of the code-based `authorize` flow.  Do we need it called out separately?